torch/distributed/fsdp/_init_utils.py

```diff
@@ -1042,7 +1042,11 @@ def _get_compute_device(
     if device_from_device_id is not None and compute_device != device_from_device_id:
         raise ValueError(
             f"Inconsistent compute device and `device_id` on rank {rank}: "
-            f"{compute_device} vs {device_from_device_id}"
+            f"{compute_device} vs {device_from_device_id}."
+            "If this error is related to meta device, please ensure"
```

Comment:
Was `compute_device` the one that was `torch.device("meta")`?

If so, would it make more sense to only add this suffix to the error message in that case specifically?